src/jmh/java/net/sf/jabref/benchmarks/Benchmarks.java

```diff
@@ -44,7 +44,7 @@ public void init() throws IOException, SaveException {
         Random randomizer = new Random();
         for (int i = 0; i < 1000; i++) {
             BibEntry entry = new BibEntry();
-            entry.setCiteKey("id" + i);
+            entry.setBibTeXKey("id" + i);
```

Comment:
Hm, what is your problem with getCiteKey or getKey and getID? The description makes it clear what are the differences. Also the entry is a BibEntry, so so why clutter the method name with BibTeX again? It's not a big deal, but I think a crisper method name is nicer here.